source/documentation/fuseki2/__index.md

```diff
@@ -35,6 +35,7 @@ transactional persistent storage layer, and incorporates
     - [As a Docker container](fuseki-main#fuseki-docker)
     - [As an embedded SPARQL server](fuseki-embedded.html)
     - [Security and data access control](fuseki-data-access-control.html)
+    - [Fuseki Server Protocol (REST API)](fuseki-server-protocol.html)
```

Comment:
This only applies to fuseki-webapp but putting it here implies it applies to both forms.

See 1d3953a1 - it has been incomplete for a long while.

```
    [Server Admin Protocol](fuseki-server-protocol.html)
```

It may not be the way to have admin for fuseki-main, and quite possibly that would only have some of the functionality. To create space for Fuseki to evolve, at least only put in the fuseki-webapp section.

Reply:
TBH I struggle to understand the relationship between the many Fuseki modules these days, do we have any documentation that actually explains that because even as someone relatively familiar with the code base I find this confusing

Reply:
The pages were reorged 2 months ago - there is a page for [Fuseki/webapp](https://jena.apache.org/documentation/fuseki2/fuseki-webapp.html) and a page for [Fuseki/main](https://jena.apache.org/documentation/fuseki2/fuseki-main.html), which includes docker, and for Fuseki+GeoSPARQL.

There are traditional ways to run services and modern ones, not that we see many instances of multiple webapps in a single Tomcat installation these days.

The many modules are a maven effect and, yes, there are quite a few of them. Each module does one step, so while one or two steps could be combined into one module, by the time we have core code, webapp, command line, different packagings, it all mounts up. Might be slightly easier in gradle but we get the release process from Apache parent POM.

I can add a page for the module dependencies if that helps.